Gemfile

```diff
@@ -5,4 +5,7 @@ git_source(:github) do |repo_name|
   "https://github.com/#{repo_name}.git"
 end
 
+gem 'danger'
 gem 'rubocop', '0.57.2', require: false
+gem 'danger-checkstyle_format'
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/5d0b2ccf5e054074d74223f40c537788a7a13706/Gemfile#L10"></span>Bundler/OrderedGems: Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem <code>danger-checkstyle_format</code> should appear before <code>rubocop</code>.</td>
    </tr>
  </tbody>
</table>